app/src/main/java/com/dimagi/biometric/fragments/BaseMatchFragment.java

```diff
@@ -71,6 +70,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return view;
     }
 
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        Bundle args = getArguments();
+        String errors = null;
+        if (args != null) {
+            errors = args.getString("errors");
+        }
```

Comment:
Same here, can we refactor this

Reply:
[71a915d](https://github.com/dimagi/biometric-android/pull/3/commits/71a915d5adbdedb8a00e085a5796b687e9903e21)